_pygeoapi_process/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
_pygeoapi_process/calc_weights_catalog.py,sha256=MH6PQ4vXdsnSLHvrscyxl2QRofzLDFM93hnYKvHhh4I,6028
_pygeoapi_process/run_weights_catalog.py,sha256=qgAOYBiuQPhC6ajGq6PtOW9Z6WKaif_bw79JWD-i0Ts,8050
gdptools_pygeoapi_plugin-0.0.2.dev3.dist-info/entry_points.txt,sha256=ziaPZaPKDm4FsSEUyxVHEYZVhrUK6cfMqu30cp3EwOI,83
gdptools_pygeoapi_plugin-0.0.2.dev3.dist-info/LICENSE.md,sha256=DBEMgl2KkjOxhLtUQP6eEne-wZe6EyFeWuQgLZ3h4dY,1640
gdptools_pygeoapi_plugin-0.0.2.dev3.dist-info/WHEEL,sha256=DA86_h4QwwzGeRoz62o1svYt5kGEXpoUTuTtwzoTb30,83
gdptools_pygeoapi_plugin-0.0.2.dev3.dist-info/METADATA,sha256=5ZuIV916k-wZG86tvvDB0Z8cTkIm2pQUphAvcBe-kJk,3492
gdptools_pygeoapi_plugin-0.0.2.dev3.dist-info/RECORD,,
